src/App.js

```diff
@@ -1,19 +1,146 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
-
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { getTodosWithUsers } from './getTodos';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Static list of todos</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-}
+export class App extends Component {
+  state = {
+    newTodos: [...todos],
+    newUsers: [...users],
```

Comment:
исходные файлы можно назвать `todosFromServer` `usersFromServer`, а в стейте без new
```suggestion
    todos: [...todosFromServer],
    users: [...usersFromServer],
```

Reply:
змінив, питання, а чому не можна відразу писати
todos: [...todos],???

Reply:
і змінив ось так, бо інакше еслінт каже, що в глобальному скопі вже users and todos вже є.
```suggestion
    todosFromServer: [...todos],
    usersFromServer: [...users],
```